cmd/agent/daemon/app/app.go

```diff
@@ -530,3 +562,32 @@ func waitWithTimeout(errg *errgroup.Group, timeout time.Duration) error {
 		return err
 	}
 }
+
+func setupStorageMetrics(metricsClient custommetrics.MetricClient) (pipeline.BlockDeviceMetricsWriter, pipeline.FilesystemMetricsWriter, error) {
+	blockDeviceMetrics, err := pipeline.NewBlockDeviceMetricsWriter(metricsClient)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create block device metrics writer: %w", err)
+	}
+
+	filesystemMetrics, err := pipeline.NewFilesystemMetricsWriter(metricsClient)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create filesystem metrics writer: %w", err)
+	}
+
+	return blockDeviceMetrics, filesystemMetrics, nil
+}
+
+func createMetricsClient(cfg *config.Config) (custommetrics.MetricClient, error) {
+	if !cfg.Castai.Valid() {
+		return nil, fmt.Errorf("cast config is not valid")
+	}
+
+	metricsClientConfig := custommetrics.Config{
+		APIAddr:   cfg.Castai.APIGrpcAddr,
```

Comment:
Now this points to `kvisor.prod-master.cast.ai` The issue is that this service doesn't have custom metrics gRPC endpoints. We have `temelemtry.prod-master.cast.ai` for that. 
I see few options:
1. Replace prefix `kvisor` with `telemetry` for this client. Easy.
2. Expose a custom metrics server as a proxy inside our internal kvisor server. More complex.